### PR TITLE
[DPC-4434] Bump junit.jupiter to 5.11.3

### PR DESCRIPTION
**Pull request by MEspositoE14s**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4434

## 🛠 Changes

Bump junit-jupiter to 5.11.3 to get our integration tests running again.

## ℹ️ Context

When running `make ci-app`, both locally and as part of the Github build process our tests haven't been running.  This was because the most recent bump of Drop Wizard brought in a transitive dependency on junit-jupiter 5.11.3, and we were running 5.10.2.  This conflict doesn't cause a failure, but it does prevent Junit from finding any of our tests.  This PR bumps our version to match the one pulled in from Drop Wizard and gets our tests running again.

### Note

This change was merged on 11/26 and our tests haven't been running ever since.  We're going to have to go back and fix any potential failing tests we've introduced since then.

## 🧪 Validation

- Ran make ci-app locally and tests are running.  
- Submitted this PR and can see the tests failing.

### CODE REVIEW

**Comment by jdettmannnava** [comment]
Aren't we going to need to fix the tests in order to merge?

**Comment by MEspositoE14s** [comment]
> Aren't we going to need to fix the tests in order to merge?

Yep, and in the meantime we should probably stop merging new code until it's done.

File: dpc-common/src/main/java/gov/cms/dpc/common/logging/filters/SecretLoggingFilter.java [hunk 1]
```diff
@@ -33,12 +34,12 @@ public SecretLoggingFilter() {
 	@Override
 	public Filter<ILoggingEvent> build() {
 		// Clean the secrets list
-        secrets.removeIf(secret -> !envVars.containsKey(secret));
+        List<String> containsSecrets = secrets.stream().filter(envVars::containsKey).collect(Collectors.toList());
```
Comment (MEspositoE14s) [inline dpc-common/src/main/java/gov/cms/dpc/common/logging/filters/SecretLoggingFilter.java:37]: Still an improvement over the iterator!

File: dpc-api/pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <mainClass>gov.cms.dpc.api.DPCAPIService</mainClass>
-        <jjwt.version>0.12.6</jjwt.version>
+        <jjwt.version>0.11.4</jjwt.version>
```
Comment (ashley-weaver) [inline dpc-api/pom.xml:17]: reverting this to the version we had before, as there's some signing key issues in JWTUnitTests that will take some refactoring to resolve

**Comment by lukey-luke** [comment]
JJWT changes basically undo most of the changes from [THIS PR](https://github.com/CMSgov/dpc-app/pull/2351/files). `make ci-app` is passing LGTM

**Review by lukey-luke — APPROVED** [review]

**Pull request merged**